server/src/types/song.types.ts

```diff
@@ -0,0 +1,21 @@
+import { Types } from 'mongoose';
+import { MongoInjectedType } from './mongo.types';
+
+type SongSchema = {
+  title: string;
+  tempo: Types.Array<string>;
+  originalKey: string;
+  tags: Types.Array<string>;
+  artist: string;
+  firstLine: string;
+  id: Types.ObjectId;
```

Comment:
you dont need to define `id` because `_id` is autogenerated by mongo. please delete